experimental/helm/charts/pipelines/templates/cache/rolebinding.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if .Values.cache.enabled }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  labels:
+    {{- include "kubeflow-pipelines.cacheLabels" . | nindent 4 }}
+  name: kubeflow-pipelines-cache-binding
+  namespace: {{ include "kubeflow-pipelines.namespace" . }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: kubeflow-pipelines-cache-role
+subjects:
+- kind: ServiceAccount
+  name: {{ include "kubeflow-pipelines.cache.serviceAccountName" . }}
```

Comment:
Are you sure that KFP does not break if we change that? Please only make things configurable that are actually configurable.